### PR TITLE
Ga workshop

### PR DESCRIPTION
**Pull request by Sambillingham**
Content for GA Workshop - replaces coming soon.

screen shot 2015-11-03 at 15 36 11
screen shot 2015-11-03 at 15 36 16

### CODE REVIEW

File: sass/partials/_sessions.scss [hunk 1]
```diff
@@ -41,8 +41,8 @@
   color: #fff;
 }
 // Every other speaker white  (overides 6 form above)
-.session:nth-of-type(2n+2),
-.session:nth-of-type(2n+2):before {
+.session:nth-of-type(2n+2):not(.session--workshop),
```
Comment (Sambillingham) [inline sass/partials/_sessions.scss]: These selectors are pretty unfriendly at this point as the speakers/workshops now share very little styling

File: index.html [hunk 2]
```diff
@@ -140,7 +140,16 @@ <h3 class="session__name session__name--workshop">{{js.name}}</h3>
     </div>
   </a>
 
-  {% include workshop-coming-soon.html %}
+  {% assign ga = site.data.workshops | where:"name", "General Assembly" %}
```
Comment (Sambillingham) [inline index.html]: Lots of code duplication in here now but slightly hesitant to loop it while workshops/coming soon blocks are keep changing.

**Comment by Sambillingham** [comment]
Almost forgot I actually need to merge it :)

**Comment by alicemollon** [comment]
Looks good Sam! Seems to go to take you to the wrong place on the schedule though. Takes you to the talks rather than the workshops

**Comment by ameliahumfress** [comment]
I noticed that as well! Otherwise looks great. :)


—
Founder & CEO, Steer


Twitter: @ameliahumfress

**Comment by samjbmason** [comment]
I have fixed the issue mentioned above in PR #117

**Pull request merged**